micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientConfiguration.groovy

```diff
@@ -35,6 +37,9 @@ import java.util.concurrent.ThreadFactory
 @EnableServiceDiscovery
 class ServiceRestClientConfiguration {
 
+    @Value('${microservice.restclient.connectionTimeout:-1}') int connectionTimeoutMillis
```

Comment:
OK, sounds fair.

One more thing, [here](https://github.com/4finance/micro-infra-spring/pull/175/files#diff-6cc9c8bfe82f8c4ebddf18c337e1e040R40) we introduce different naming convention for `ServiceRestClient` parameters. It's compatible with `rest.api...` Swagger config properties. Do you mind if I change this to `rest.client.connectionTimeout`?